tests/performance-test/deploy/performance-test-tg.sh

```diff
@@ -1,21 +1,93 @@
 #!/bin/bash
 # Sets up the envirovnment for performance testing
+
+#set -e
+
+oc project openshift-monitoring
+OCP_PROM_HOST=$(oc get routes --field-selector metadata.name=prometheus-k8s -o jsonpath="{.items[0].spec.host}")
+OCP_DATASOURCE=$(oc get secret -n openshift-monitoring grafana-datasources -o jsonpath='{.data.prometheus\.yaml}' \
+                | base64 -d)
+OCP_DATASOURCE=$(echo $OCP_DATASOURCE | python -c "import sys, json; source=json.load(sys.stdin)['datasources'][0]; \
+                source['url']='https://$OCP_PROM_HOST'; source['name']='OCPPrometheus'; print(json.dumps(source))")
+
+oc project sa-telemetry
+GRAF_HOST=$(oc get routes --field-selector metadata.name=grafana -o jsonpath="{.items[0].spec.host}")
+SAF_PROM_HOST=$(oc get routes --field-selector metadata.name=prometheus -o jsonpath="{.items[0].spec.host}")
+
+SAF_DATASOURCE=$(cat << EOF
+    {
+        "name"  :   "SAFPrometheus",
+        "type"  :   "prometheus",
+        "url"   :   "http://$SAF_PROM_HOST",
+        "access":   "direct",
+        "basicAuth" :false,
+        "jsonData": {
+            "timeInterval"  :   "1s"
+        }
+    }
+EOF
+)
+
 TG_CONFIGFILE=${TG_CONFIGFILE:-./config/test-configs.yml}
 
 oc delete configmap/saf-performance-test-collectd-config \
    configmap/saf-performance-test-entrypoint-script \
    configmap/saf-performance-test-configs \
    configmap/saf-performance-test-hosts \
+   configmap/grafana-dashboard-template \
```

Comment:
I feel like some of the naming here should be more specific and call out that it's `perfscale-dashboard-template` or something?

Reply:
I'm just thinking about future dashboarding work where a generic name like this could potentially cause a conflict.

Reply:
Valid point. It should be more uniquely identifiable for its use case